src/test/java/teammates/test/cases/browsertests/StudentFeedbackSubmitPageUiTest.java

```diff
@@ -167,6 +167,9 @@ private void testSubmitAction() throws Exception {
         submitPage.fillResponseTextBox(18, 0, 0, "90");
         submitPage.fillResponseTextBox(18, 0, 1, "10");
 
+        submitPage.fillResponseTextBox(19, 1, 0, "200");
+        assertEquals(submitPage.getResponseTextBoxValue(19, 0, 0), "0");
```

Comment:
Add comments to each code block.
`// total sums up to expected value, verify empty entries are filled with 0`
`// delete an auto-filled 0, verify it's auto-filled again`

Actually, you can also add another test
`// after auto-filled with 0, modify a non-zero value and remove a 0, -> verify no longer auto-filled with 0`